src/mantarray_desktop_app/constants.py

```diff
@@ -236,6 +236,9 @@
 SERIAL_COMM_MAGIC_WORD_BYTES = b"CURI BIO"
 SERIAL_COMM_TIMESTAMP_LENGTH_BYTES = 8
 SERIAL_COMM_CHECKSUM_LENGTH_BYTES = 4
+SERIAL_COMM_MAX_PACKET_LENGTH_BYTES = (
```

Comment:
does this include the magic word? or is it specifically supposed to exclude the magic word in what this is defining as the "length"?

Reply:
This includes the magic word